witness/src/Dscp/Snowdrop/Configuration.hs

```diff
@@ -141,11 +141,14 @@ type CanVerifyPayload txid payload =
         (txid, PublicKey, payload)
 
 
-type PersonalisedProof txid  payload =
-    WithSignature
+data PersonalisedProof txid payload = PersonalisedProof
```

Comment:
So this seems to be a proof for tx, right? Why `ppFee` is `Word32`? Can we rename it into something better or at least add a comment?

Reply:
It is `Word32` because I wanted some non-negative thing.

Reply:
But `Coin` is `Word64`.